webgpu/lessons/webgpu-vertex-buffers.md

```diff
@@ -744,7 +750,10 @@ We also need to update the per instance data.
   }
 ```
 
-The layout for the per instance data is like this
+Notice we queue'd the *second* view (`F32`) of the typed array, 
```

Comment:
I don't understand this comment. We haven't queue'd anything. queue'd isn't word either it's 'queued' and no idea what `F32` and `U8` are here. I think it's fine as it was.

Reply:
`staticVertexValuesF32` and `staticVertexValuesU8`, w reference to the `device.queue.writeBuffer` on line 753 (of new commit). just emphasizing view behavior, but its discussed plenty in [memory layout article](https://webgpufundamentals.org/webgpu/lessons/webgpu-memory-layout.html)

Reply:
[resolution commit](./154/commits/6b21a7057fa8547610784c0e7ad1214432ff8383)